.github/workflows/bandit_sonar.yaml

```
@@ -18,61 +18,51 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Generate Bandit skips from Ruff
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        run: pip install bandit[toml] tomli
```

Do we need `tomli`? Looks like this workflow is running with Python 3.11 which has [`tomllib`](https://docs.python.org/3/library/tomllib.html) already. Isn't `tomllib` good enough for what we need to do here?

```suggestion
        run: pip install bandit[toml]
```